### ISSUE TITLE
Entire site down now?

### ISSUE THREAD

**Issue by davisable** [minotar/imgd#162]
None of the avatar images are loading now and the minotar website is now returning a "Parked at Namecheap" page.

Did the domain name expire or something?

**Comment by ApfelUser** [comment]
@davisable not here. Germany / Frankfurt

**Comment by LukeHandle** [comment]
From WHOIS: `Expiration Date: 07-apr-2015`

Needs renewal I assume, it may continue to work until the parked name servers propagate,but we're waiting on @clone1018 to wake up.

**Comment by jk-5** [comment]
Adding the following line to /etc/hosts will fix it temporarilly:

```104.24.113.248 minotar.net```

**Comment by clone1018** [comment]
Paid for and set to auto renewal

**Comment by jk-5** [comment]
Nice. Thanks for your awesome service.

**Comment by kamaln7** [comment]
Thanks :smile:
